modules/ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,216 @@
+// TELCODOCS-272 4.8 Using Assisted Installer to Install & Manage ocp Clusters on Bare Metal
+//
+//
+//
```

Comment:
need an id

Reply:
Added ID [id=ztp-ai-install-ocp-clusters-on-bare-metal_{context}"]

Reply:
That needs an opening quote after =.

Reply:
@ktothill Here's an example of  a heading comment. It should have the path(s) to the assembly that includes the module. 

// Module included in the following assemblies:
//
// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc

[id="additional-install-config-parameters_{context}"]
= Additional `install-config` parameters